code/go/internal/spec_test.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import (
+	"testing"
+
+	_ "github.com/elastic/package-spec/code/go/internal/spec"
```

Comment:
nit: you can sort these imports

Reply:
Thanks for catching this. I need to set this up in my IDE!

Reply:
According to my IDE (goland) these imports are already sorted. I deliberately tried reordering them and my IDE re-sorted them into their current position. 🤔

Reply:
Imports should be sorted now.